pkg/controller/engine.go

```diff
@@ -73,6 +94,13 @@ func parseResouce(obj parsedK8sObjects) ([]common.Resource, []common.Service) {
 			}
 			res.Resource.FilePath = obj.ManifestFilepath
 			links = append(links, res)
+		} else if p.GroupKind == "ConfigMap" {
+			res, err := analyzer.ScanK8sConfigmapObject(p.GroupKind, p.RuntimeObject)
+			if err != nil {
+				zap.S().Errorf("error scanning Configmap object: %v", err)
+				continue
+			}
+			configMaps = append(configMaps, res)
```

Comment:
maybe convert if-else chain to a switch statement

Reply:
done